### ISSUE TITLE
Улучшения грида.

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1146]
Originally reported on Google Code with ID 1146

```
Почти все современные гриды обзовелись новыми возможностями. Перечислю то, 
что по-моему не хватает в гриде в Гедымине.
1. Возможность создание многострочных заголовков, для группировки колонок 
по каким-либо признакам. В гриде добавляется property которое 
устанавливает использование\неиспользование многострочных заголовков. 
Далее в наименование колонок определяем символ разделения (например '|') и 
если он указан, то автоматически формируется верхний заголовок.
2. Возможность работы с булевыми полями. Сейчас у нас можно задать value 
list, но так как по умолчанию у нас не стоит отображение нулей в гриде, то 
выходит не очень красиво. И вообще можно было бы добавить признак именно 
на колдонку, отображать ли нули, например в системе не надо отображать 
нули в гридах не надо, но для некоторых колонок надо. Сейчас это решить 
невозможно.
3. Можно было сформировать автоматический перенос слов в наименовании 
колонки, с автоматическим увеличением высоты её. Сейчас если наименование 
слишком длинное, то у нас просто отрезается его представление, опять же 
очень не удобно. Можно было бы регулировать через property переносить 
заголовки или нет.
4. Ну и визуальное представление.

Кто что может добавить, раскритиковать?. Смотрите прикрепление.
```

Reported by `Alexander.GoldenSoft` on 2008-12-12 08:37:07

<hr>
- _Attachment: [123.JPG](https://storage.googleapis.com/google-code-attachments/gedemin/issue-1146/comment-0/123.JPG)_

**Comment by gsbelarus** [comment]
```
Можно еще сделать сворачиваемые группы колонок, раскидать по группам все поля, и
свернуть неиспользуемые. Тогда не придется лазить в мастер установок чтобы отбразить
какое-либо поле, развернул, посмотрел, снова свернул.
В прикреплении свернутые группы выделены темным цветом.
```

Reported by `flakekun` on 2008-12-12 16:52:04

<hr>
- _Attachment: [add.jpg](https://storage.googleapis.com/google-code-attachments/gedemin/issue-1146/comment-1/add.jpg)_

**Comment by gsbelarus** [comment]
```
Заголовки в несколько строк да еще и в несколько уровней хорошо только при первом
знакомстве с гридом. Потом они только будут отъедать полезное пространство на экране.
Мне кажется что лучше иметь возможность определить несколько настроек для грида и
быстро переключаться между ними.
```

Reported by `gs1994` on 2008-12-16 14:48:37

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2014-01-09 13:35:09
- **Status changed**: `Duplicate`
- **Merged into**: #2950
